internal/node_launcher.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
is this change needed?

Reply:
At some point I needed this change to run on windows, but we should double check if it's still needed.

Reply:
confirmed, it's still needed.